labs/byzantine-api-gateway.md

```diff
@@ -0,0 +1,26 @@
+# Lab Name
+_byzantine-api-gateway_
+
+# Short Description
+_Enables client access to Hyperledger Fabric blockchain network through restful API's._
+
+# Scope of Lab
+_Currently client applications can access HLF network Peers using the GoLang, Node, or Java SDK's. This project provides a generalized API gateway that will allow users to query, execute transactions along with querying block storage and configuration using restul API's.
+
+This project provides a ready to use solution that developers would otherwise have to implement when trying to provide web clients access to a HLF network._
+
+# Initial Committers
+_Enter the Github IDs for the set of initial committers._
+- https://github.com/dpitt
+- https://github.com/jaimeniswonger
+
+
+# Sponsor
+_Provide the name of your sponsor. A sponsor is required and must be a maintainer of one of the Hyperledger projects, a TSC member, or a WG chair._
+- https://github.com/<user_id> or Name (email@example.org) - Role (e.g., "Chair of the XXX workgroup")
+
+  Don't have one yet, looking for one 
```

Comment:
You can add me as a sponsor- looks like a worthwhile project
vip@dlt.nyc chair of Identity WG

Reply:
Thanks! wiil do..